scripts/lib-franklin.js

```diff
@@ -144,6 +144,11 @@ export function decorateIcons(element = document) {
       } else {
         span.innerHTML = iconHTML;
       }
+      const a = span.closest('a');
+      if (a && !a.title) {
+        const iconName = span.classList[1];
+        a.title = iconName.substring(5) || `${a.href}`;
```

Comment:
I know `5` is meant to represent `icon-` but this has maintenace issues since someone 6 months from now may now know what `5` means. Could you just extract what you want from `span.className` using a RegEx?

Reply:
If you need to add a comment to explain what the code is doing that is a sign that the code is probably not very good.